package.json

```diff
@@ -50,7 +50,7 @@
     "apis-helpers": "0.0.1"
   },
   "scripts": {
-    "test": "node_modules/mocha/bin/mocha test/integration",
+    "test": "mocha test/integration",
```

Comment:
What about running ./test and do some checking if mocha is globally available, else default to the old one with correct seperator for windows and linux?